### PR TITLE
[ci skip]add Chinese comments for DrawNode

### PR DESCRIPTION
**Pull request by zilongshanren**
This PR also changes some English comments of v3-doc, we should sync them to v3.

### CODE REVIEW

**Comment by zilongshanren** [comment]
please @pandamicro review my translation.
Thanks.

File: cocos/2d/CCDrawNode.h [hunk 1]
```diff
@@ -44,253 +44,417 @@ class PointArray;
...
+     * 
+     * @param point @~english A Vec2 point.
+     * @~chinese 一个Vec2类型的点
+     * @param pointSize @~english The point size.
+     * @~chinese 点的大小
+     * @param color @~english The point color.
+     * @~chinese 点的颜色
      * @js NA
      */
+
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h]: Please check all useless blank lines

File: cocos/2d/CCDrawNode.h [hunk 2]
```diff
@@ -44,253 +44,417 @@ class PointArray;
...
     void drawLine(const Vec2 &origin, const Vec2 &destination, const Color4F &color);
     
-    /** Draws a rectangle given the origin and destination point measured in points.
+    /** @~english Draws a rectangle given the origin and destination point measured in points.
      * The origin and the destination can not have the same x and y coordinate.
      *
-     * @param origin The rectangle origin.
-     * @param destination The rectangle destination.
-     * @param color The rectangle color.
+     * @~chinese 给定一个起始点坐标和目的地点坐标画一个矩形，坐标的单位是“Point”
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h]: No description about what this API does. It draws a rect

File: cocos/2d/CCDrawNode.h [hunk 3]
```diff
@@ -44,253 +44,417 @@ class PointArray;
...
-    /** Draws a polygon given a pointer to point coordinates and the number of vertices measured in points.
+    /** @~english Draws a polygon given a pointer to point coordinates and the number of vertices measured in points.
      * The polygon can be closed or open.
      *
-     * @param poli A pointer to point coordinates.
-     * @param numberOfPoints The number of vertices measured in points.
-     * @param closePolygon The polygon can be closed or open.
-     * @param color The polygon color.
+     * @~chinese 给定一个指向一组点坐标的指针和坐标点数量来绘制多边形
+     * 多边形可以闭合，也可以不闭合。
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h:142]: Should mention whether it fill the polygon or not

File: cocos/2d/CCDrawNode.h [hunk 4]
```diff
@@ -44,253 +44,417 @@ class PointArray;
...
      *
-     * @param center The circle center point.
-     * @param radius The circle rotate of radius.
-     * @param angle  The circle angle.
-     * @param segments The number of segments.
-     * @param drawLineToCenter Whether or not draw the line from the origin to center.
-     * @param scaleX The scale value in x.
-     * @param scaleY The scale value in y.
-     * @param color Set the circle color.
+     * @~chinese 给定圆的中心,半径和线段数量来绘制一个圆。
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h]: fill or not

File: cocos/2d/CCDrawNode.h [hunk 5]
```diff
@@ -44,253 +44,417 @@ class PointArray;
...
     
-    /** Draws a rectangle with 4 points.
+    /** @~english Draws a rectangle with 4 points.
      *
-     * @param p1 The rectangle vertex point.
-     * @param p2 The rectangle vertex point.
-     * @param p3 The rectangle vertex point.
-     * @param p4 The rectangle vertex point.
-     * @param color The rectangle color.
+     * @~chinese 画一个包含4个顶点的矩形。
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h]: The order should be mentioned, first is left top ? second is right top ?

File: cocos/2d/CCDrawNode.h [hunk 6]
```diff
@@ -44,253 +44,417 @@ class PointArray;
...
     void drawPolygon(const Vec2 *verts, int count, const Color4F &fillColor, float borderWidth, const Color4F &borderColor);
 	
-    /** draw a triangle with color. 
+    /** @~english draw a triangle with color. 
      *
-     * @param p1 The triangle vertex point.
-     * @param p2 The triangle vertex point.
-     * @param p3 The triangle vertex point.
-     * @param color The triangle color.
+     * @~chinese 使用给定颜色绘制一个三角形。
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h]: Fill or not?

File: cocos/2d/CCDrawNode.h [hunk 7]
```diff
@@ -44,253 +44,417 @@ class PointArray;
...
 
-    /** draw a quadratic bezier curve with color and number of segments, use drawQuadBezier instead.
+    /** @~english draw a quadratic bezier curve with color and number of segments, use drawQuadBezier instead.
      *
-     * @param from The origin of the bezier path.
-     * @param control The control of the bezier path.
-     * @param to The destination of the bezier path.
-     * @param segments The The number of segments.
-     * @param color The quadratic bezier color.
+     * @~chinese 给定线段颜色和线段数量画一条二次贝塞尔曲线. 该函数已不推荐使用，使用drawQuadBezier代替。
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h:407]: Quadratic 四次

File: cocos/2d/CCDrawNode.h [hunk 8]
```diff
@@ -298,11 +462,11 @@ class CC_DLL DrawNode : public Node
     /**
      * @js NA
      */
-	void onDraw(const Mat4 &transform, uint32_t flags);
+  void onDraw(const Mat4 &transform, uint32_t flags);
```
Comment (pandamicro) [inline cocos/2d/CCDrawNode.h]: Four space tab

**Comment by pandamicro** [comment]
Review done, please check @andyque

File: cocos/2d/CCDrawNode.h (hunk 7 above)
Comment (zilongshanren) [inline cocos/2d/CCDrawNode.h:407]: Quadratic 是2次啊，你查下字典 4次是quartic

File: cocos/2d/CCDrawNode.h (hunk 5 above)
Comment (zilongshanren) [inline cocos/2d/CCDrawNode.h]: yes, I  should also modify the original english comments.

Comment (zilongshanren) [inline cocos/2d/CCDrawNode.h]: @pandamicro  I have seen the source code, no particular order is needed here.

**Comment by zilongshanren** [comment]
@pandamicro  modification done, please check.

**Pull request merged**